assets/pages/contributions-landing-us/contributionsLandingUS.jsx

```diff
@@ -43,9 +43,8 @@ const content = (
       <Contribute
         copy={
           <span>
-            Contributing to The&nbsp;Guardian makes a big impact. If you&#39;re able,
-            please consider <strong>monthly</strong> support - it will help to
-            fund our journalism for the long term.
+            Make a monthly commitment to support The Guardian long-term or a one-time contribution
+            as and when you feel like it – choose the option that suits you best.
```

Comment:
There's no markup in this so I think you can pass it in as a string, as the copy prop accepts `string | Node`. The wrapping `<span>` was previously only there because of the `<strong>monthly</strong>`.